ckanharvester.py

```diff
@@ -0,0 +1,63 @@
+ # -*- coding: utf-8 -*-
+import urllib2
+import httplib
+import datetime
+import socket
+from sqlalchemy import create_engine
+from sqlalchemy import text
+from sqlalchemy import Table, MetaData
+from ckanext.harvest.model import HarvestJob, HarvestObject, HarvestGatherError
+import logging
+
+log = logging.getLogger(__name__)
+
+from base import HarvesterBase
+
+#harvesteris parašytas pakeitus ckan-harvester kodą
+#kol kas harvestina viską kas yra duombazėj.
+#kadangi pas mane nesukonfiguruotas cron tai pats harvesteris automatiškai nepasileidžia
+#norint ckan privesti paleisti harvesti reik rašyti šias komandas esant opendata.gov.lt/ckan
+#paster --plugin=ckanext-harvest harvester gather_consumer vienam terminalo lange ir
+#paster --plugin=ckanext-harvest harvester fetch_consumer kitam terminalo lange
+
+class CKANHarvester(HarvesterBase):
```

Comment:
Class name is misleading, we are no harvesting from CKAN, we ar harvesting from a legacy opendata.gov.lt database. I would suggest to use `ODGovLt` code name.